libs/rest/generated-rest-client/src/main/kotlin/net/corda/restclient/dto/UpdateConfigParametersString.kt

```diff
@@ -0,0 +1,11 @@
+package net.corda.restclient.dto
+
+import net.corda.libs.configuration.endpoints.v1.types.ConfigSchemaVersion
+
+/**
+ * This is a copy of [net.corda.libs.configuration.endpoints.v1.types.UpdateConfigParameters] as the config as [net.corda.rest.JsonObject]
+ * is not handled correctly. See [JIRA comment](https://r3-cev.atlassian.net/browse/ES-2162?focusedCommentId=303404)
+ */
+data class UpdateConfigParametersString(
+    val section: String, val version: Int, val config: String, val schemaVersion: ConfigSchemaVersion
```

Comment:
`val config: String` - may not work for all the use cases. Specifically I am thinking of the situation when config content will be passed as embedded Json object as part of the payload. Please see in documentation: https://docs.r3.com/en/platform/corda/5.2/deploying-operating/config/dynamic.html

Reply:
The example you've linked to is setting config values by specifying a nested json object.
That's exactly what the CLI Network plugin > BaseOnboard > configureGateway function is doing. Once we have the json object created, we can serialise it into a string value, to pass into the request payload.
https://github.com/corda/corda-runtime-os/blob/c2db9e26fc2535fdbdf4d5570e8cc955a5b65af7/tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/BaseOnboard.kt#L261-L274

Reply:
Yes, that would work, but we have to call `json.writeValueAsString()` every time we try to update config. Also, "on the wire" we will have escaped JSON in this case, which IMO not as natural to inspect as embedded JSON.
Since configuration section will *always* be JSON, how about changing to:
```suggestion
    val section: String, val version: Int, val config: ObjectNode, val schemaVersion: ConfigSchemaVersion
```
?
Of course, this would introduce dependency on Jackson library here. If this is a problem, though, we can settle to `String` as a common denominator.

Reply:
Object node appears to work, tested fine. Pushing change and updating ES-2382 to reflect new name